trunk/src/lsc/mysqldef.py

```diff
@@ -217,8 +217,10 @@ def ingestredu(imglist,force='no',dataredutable='photlco',filetype=1):
             print img,database
             lsc.mysqldef.deleteredufromarchive(string.split(img,'/')[-1],dataredutable)
             print 'delete line from '+str(database)
-            exist=lsc.mysqldef.getfromdataraw(conn,dataredutable,'filename', string.split(img,'/')[-1],column2='filename')
-
+            exist = ()
+#            exist=lsc.mysqldef.getfromdataraw(conn,dataredutable,'filename', string.split(img,'/')[-1],column2='filename')
+#            print(exist)
```

Comment:
I would just remove this code. It's in github so why leave it as a comment?

Reply:
sure you can remove it.